### DISCUSSION TITLE
Simple Navigation - Jan 16

### DISCUSSION THREAD

**Discussion by chaitanyaj2222** [RealDevSquad/Centauri-React-Native-Cohort#17] [Show and tell]
Were you able to get it working? Post a screen recording

**Comment by chaitanyaj2222** [comment]
https://user-images.githubusercontent.com/56331036/149864042-181adb29-92b6-4cac-8095-0bc34566aef9.mp4

**Comment by prachic17** [comment]
Navigation Task

https://user-images.githubusercontent.com/61531836/149904262-a431b44f-9832-40b9-a946-1b222e531f7e.mp4

**Comment by aakankshadhurandhar** [comment]
https://user-images.githubusercontent.com/34390765/149950799-be35c9a6-b3ec-4cba-a916-8fe9b60c9d21.mp4

**Comment by Sahil-Bhatia** [comment]
https://user-images.githubusercontent.com/31650286/150096516-70729010-95da-4ac8-9b86-dae6247466a9.mp4

**Comment by mohankumar80** [comment]
https://user-images.githubusercontent.com/50794740/151552303-46d78c90-271c-452a-b617-59062266f0aa.mp4

**Comment by ParakhBansal** [comment]
https://user-images.githubusercontent.com/65717231/151776654-6716ec80-ff21-4333-b5cd-3e5a9beaaa5b.mp4

**Comment by jatin33** [comment]
Git repo: https://github.com/jatin33/RDS-react-native-navigation-task-2
https://user-images.githubusercontent.com/13780467/149660492-f4d770ff-2f45-4a37-a863-2c96c553c26f.mov
